pkgs/development/tools/icr/default.nix

```
@@ -1,15 +1,26 @@
-{ stdenv, lib, fetchFromGitHub, crystal, shards, makeWrapper, pkg-config, which
-, openssl, readline, libyaml, zlib }:
+{ stdenv
+, lib
+, fetchFromGitHub
+, crystal
+, shards
+, makeWrapper
+, pkg-config
+, which
+, openssl
+, readline
+, libyaml
+, zlib
+}:
 
 crystal.buildCrystalPackage rec {
   pname = "icr";
-  version = "0.8.0";
+  version = "0.8.0.20201006";
```

```suggestion
  version = "unstable-2020-10-06";
```

If this is not a tag on github please use unstable-XXXX-XX-XX with the date of the commit.